bin/release-bom

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC1091
+set -eo pipefail
+
+help() {
+  echo "Description:"
+  echo "    Generates and publishes BOM JSON files to the public release repository."
+  echo
+  echo "Usage:"
+  echo "    ${0##*/} [-x] <version>"
+}
+
+while getopts ':hx' option ; do
+  case "${option}" in
+    h) help ; exit ;;
+    x) set -x ;;
+    *) help ; exit 1 ;;
+  esac
+done
+shift $((OPTIND - 1))
+
+if [[ "${#@}" -eq 0 ]] ; then
+  echo "ERROR: missing version argument"
+  echo
+  help
+  exit 1
+fi
+
+version="$1"
+repo_url="https://updates.astronomer.io/astronomer-software/releases"
+json_file_name="astronomer-${version}.json"
+local_json_file_path="./astronomer_bom_generator/${json_file_name}"
+max_age="300"
+
+# Step 1: Runs the Python extractor script to generate the BOM JSON file
+echo "Running the extractor script to generate ${json_file_name}..."
+python3 ./astronomer/astronomer_bom_generator/extractor.py --version "${version}"
```

Comment:
Yes it would be pip install

Reply:
I think maybe we should make the astronomer-bom-generator repo public just to make installation easy. It's not doing anything proprietary. We should probably reset the git history though just in case.

Reply:
Sure